AtLeastValidator.php

```diff
@@ -88,6 +88,9 @@ public function init()
         }
         if ($this->message === null) {
             $this->message = 'You must fill at least {min} of the attributes {attributes}.';
+            if($this->max > 0) {
```

Comment:
Could you remove the dot from the end of the L90 and make a more simple sintax, please?
Suggestion:
`$this->message .= $this->max > 0 ? ' and at most {max} attribute(s).' : '.';`
Or you could even think a better way to generate a better error message for cases when `$min=0` and `$max=2`, for example.

Thank you so much.